src/test/java/catan/controllers/play/BuildSettlementTest.java

```diff
@@ -0,0 +1,171 @@
+        String userToken3 = loginUser(USER_NAME_3, USER_PASSWORD_3);
+
+        int gameId = createNewGame(userToken1, false).path("gameId");
+        int node_id = viewGame(userToken1, gameId).path("map.nodes[0].nodeId");
+        //TODO: need to fetch gameUserId from JSON, but now it is not returned
+        //int gameUser_1_Id = viewGame(userToken1, gameId).path("gameUsers[0].id");
+
+        joinPublicGame(userToken2, gameId);
+        joinPublicGame(userToken3, gameId);
+
+        setUserReady(userToken1, gameId);
+        setUserReady(userToken2, gameId);
+        setUserReady(userToken3, gameId);
+
+        viewGame(userToken1, gameId)
+                .then()
+                .statusCode(200)
+                .body("map.nodes[0].nodeId", is(node_id))
+                .body("map.nodes[0].building", nullValue())
+                .body("status", equalTo("PLAYING"));
+
+        buildSettlement(userToken1, gameId, node_id)
+                .then()
+                .statusCode(200)
+                .body("map.nodes[0].nodeId", is(node_id))
+                .body("map.nodes[0].building.ownerGameUserId", is(1))   //need to compare with gameUser_1_Id, when it will be available
+                .body("map.nodes[0].building.built", equalTo("SETTLEMENT"))
+                .body("status", equalTo("PLAYING"));
+
+        buildSettlement(userToken2, gameId, node_id)
+                .then()
+                .statusCode(400)
+                .body("errorCode", equalTo("ERROR"));
+    }
+
+    @Test
+    public void should_fail_if_try_to_build_settlement_close_to_another_settlement_less_than_2_roads() {
+        String userToken1 = loginUser(USER_NAME_1, USER_PASSWORD_1);
+        String userToken2 = loginUser(USER_NAME_2, USER_PASSWORD_2);
+        String userToken3 = loginUser(USER_NAME_3, USER_PASSWORD_3);
+
+        int gameId = createNewGame(userToken1, false).path("gameId");
+        int node_id = viewGame(userToken1, gameId).path("map.nodes[0].nodeId");
+
+        //TODO: need to fetch gameUserId from JSON, but now it is not returned
+        //int gameUser_1_Id = viewGame(userToken1, gameId).path("gameUsers[0].id");
+
+        joinPublicGame(userToken2, gameId);
+        joinPublicGame(userToken3, gameId);
+
+        setUserReady(userToken1, gameId);
+        setUserReady(userToken2, gameId);
+        setUserReady(userToken3, gameId);
+
+        viewGame(userToken1, gameId)
+                .then()
+                .statusCode(200)
+                .body("map.nodes[0].nodeId", is(node_id))
+                .body("map.nodes[0].building", nullValue())
+                .body("status", equalTo("PLAYING"));
+
+        buildSettlement(userToken1, gameId, node_id)
+                .then()
+                .statusCode(200)
+                .body("map.nodes[0].nodeId", is(node_id))
+                .body("map.nodes[0].building.ownerGameUserId", is(1))   //need to compare with gameUser_1_Id, when it will be available
```

Comment:
we can check userId instead of gameUserId:
.body("map.nodes[0].building.buildingOwner.user.id", is(...))